components/textfield/stories/textarea.stories.js

```diff
@@ -0,0 +1,245 @@
+import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { isDisabled, isFocused, isInvalid, isKeyboardFocused, isLoading, isQuiet, isReadOnly, isRequired, isValid, size } from "@spectrum-css/preview/types";
+import { HelpTextOptionsTextArea } from "./template.js";
+import { TextFieldGroup } from "./textarea.test.js";
```

Comment:
```suggestion
import { TextFieldGroup } from "./textarea.test.js";
import metadata from "../metadata/metadata.json";
import packageJson from "../package.json";
```

I think this will keep in line with the other imports in the other components.